avsomr/ret.cpp

```diff
@@ -122,6 +122,9 @@ QString Ret::text(Code c)
             case LocalInstaling:    return QObject::tr("Local OMR engine installing…");
             case LocalFailedExec:   return QObject::tr("Failed executing local OMR engine");
             case LocalAlreadyBuilding: return QObject::tr("Local OMR engine already building…");
+
+            case AvsOmrFirst:
+            case AvsOmrLast: break;
```

Comment:
could use `default:` here, but then added cases won't get caught by a compiler warning.

Reply:
Thanks, better without default, I will be more careful